_spinnaker/pacrd.md

```diff
@@ -346,9 +346,126 @@ the resource and looking in the "Events section":
 kubectl describe pipeline myapplicationpipeline
 ```
 
+## Artifacts
+
+An [artifact](artifact-concept) is an object that references an external
+resource. Examples include a Docker container, a file in source control, an AMI,
+or a binary blob in S3. Artifacts in PaCRD come in two types:
+
+- **Definitions** contain all necessary information to locate an artifact.
+- **References** contain enough information to find a Definition.
+
+### Defining Artifacts
+
+Define your pipeline artifacts in a section called `expected artifacts`. The
+following example defines a single container image that the pipeline expects as
+an input to the BakeManifest stage:
+
+```yaml
+apiVersion: pacrd.armory.spinnaker.io/v1alpha1
+kind: Pipeline
+metadata:
+  name: my-pipeline
+spec:
+  description: A sample showing how to define artifacts.
+  application: my-application
+  expectedArtifacts:
+    - id: &image-id my-application-docker-image
+      displayName: *image-id
+      matchArtifact:
+        type: docker/image
+        properties:
+          name: my-organization/my-container
+          artifactAccount: docker-registry
```

Comment:
@dogonthehorizon - as I mentioned in a Slack PM to you, I think we need to provide more guidance here on where to find the `properties` key/value pairs. The example doesn't match info in the [Docker image](https://www.spinnaker.io/reference/artifacts-with-artifactsrewrite/types/docker-image/) artifact page on the Spinnaker docs.